### DISCUSSION TITLE
Unique constraint on multiple fields

### DISCUSSION THREAD

**Discussion by sergeyubozhenko** [keystonejs/keystone#7559] [Feature requests]
Hi!
How can I create unique constraint(PostgreSQL) on multiple fields in Keystone 6? So I need a migration like `ALTER TABLE "Course" ADD CONSTRAINT "Course_unique_user_course" UNIQUE("course", "user")
`
I didn't find anything about unique index on multiple in documentation Keystone.

**Comment by raveling** [comment]
@sergeyubozhenko thanks for raising this. It's on our radar. I've made this a feature request.

**Reply by dcousens** [reply]
Related (and a subset of) to https://github.com/keystonejs/keystone/discussions/7705
